packages/jupyterlab-manager/src/plugin.ts

```diff
@@ -74,10 +74,16 @@ const SETTINGS: WidgetManager.Settings = { saveState: false };
  * Iterate through all widget renderers in a notebook.
  */
 function* widgetRenderers(nb: Notebook) {
-  for (let cell of nb.widgets) {
+  // for (let cell of nb.widgets) {
+  for (let i = 0; i !== nb.widgets.length; ++i) {
+    let cell = nb.widgets[i];
     if (cell.model.type === 'code') {
-      for (let codecell of (cell as CodeCell).outputArea.widgets) {
-        for (let output of toArray(codecell.children())) {
+      // for (let codecell of (cell as CodeCell).outputArea.widgets) {
+      for (let j = 0; j !== (cell as CodeCell).outputArea.widgets.length; ++j) {
+        let codecell = (cell as CodeCell).outputArea.widgets[j];
+        // for (let output of toArray(codecell.children())) {
+        for (let k = 0; k !== toArray(codecell.children()).length; ++k) {
+          let output = toArray(codecell.children())[k];
```

Comment:
Same here:

Even with `"downlevelIteration": true`, typescript does not seem to properly compile `for .. of` loops to es5.